src/compiler/sys.ts

```diff
@@ -528,6 +529,8 @@ namespace ts {
               _crypto = undefined;
             }
 
+            const Buffer: typeof global.Buffer = require("buffer").Buffer;

```

Comment:
Despite the fact that we ephemerally get the node.d.ts types due to the fact we're using them along with ts-node for our gulpfile, we don't depend on actual node types anywhere else in compiler. Yes, this makes this less type-safe but I don't necessary want to take the dependency for just one declaration in the entire project.